mmctools/helper_functions.py

```diff
@@ -1353,18 +1356,22 @@ def calcVRM(hgt,window=None,footprint=None,slope_zscale=1.0,return_slope=False):
     assert len(np.shape(hgt)) == 2, 'hgt must be 2-dimensional. Currently has {} dimensions'.format(len(np.shape(hgt)))
     ny,nx = np.shape(hgt)
 
+    # Determine scale based on resolution of hgt array
+    zscale = 1/res
+
     # Get slope and aspect:
     hgt_rd = rd.rdarray(hgt, no_data=-9999)
-    rd.FillDepressions(hgt_rd, in_place=True)
-    slope  = rd.TerrainAttribute(hgt_rd, attrib='slope_riserun', zscale=slope_zscale)
+    if fill_depressions:
+        rd.FillDepressions(hgt_rd, in_place=True)
+    slope  = rd.TerrainAttribute(hgt_rd, attrib='slope_degrees',zscale=zscale)
```

Comment:
For backwards compatibility, should we make this a kwarg, with the default being `slope_riserun`? Or add a "degrees" flag and set the attrib accordingly?

Reply:
I'm not opposed to a flag (even though I see no use case), but `riserun` here would be incorrect. The computation of `rugz` and `rugdxy` depends on the slope _angle_, not rise-run ($\tan^{-1}$)

Reply:
Oh I see. Let's just leave it as is, then.